packages/design-system/src/__generated__/figma-design-tokens.ts

```diff
@@ -493,28 +535,31 @@ export const color = {
   foregroundOverwrittenFlexUi: "#dc3d43",
   foregroundInversePrimary: "#ff22ae",
   foregroundLinkNotVisited: "#006adc",
+  foregroundLinkHoverMain: "#656869",
+  foregroundLinkHoverTextSubtle: "#656869",
+  foregroundLinkHoverTextMoreSubtle: "#656869",
+  foregroundLinkHoverContrast: "#adb1b4",
```

Comment:
foregroundLinkHoverMain: "#656869",
foregroundLinkHoverTextSubtle: "#656869",
foregroundLinkHoverTextMoreSubtle: "#656869",

all have the same color

Reply:
Correct. A choice made for clarity, since there is not easy way to say that one token works for all 3 cases.